GiEnJul/ClientApp/src/components/admin/overview/Statistics.tsx

```diff
@@ -0,0 +1,88 @@
+import { Card, Grid, Typography } from "@material-ui/core";
+import React from "react";
+import { GiverType, RecipientType, SelectedConnectionType } from "./Types";
+import { FiberManualRecord } from "@material-ui/icons";
+import useStyles from "./Styles";
+
+type IStatistics = {
+  givers: GiverType[] | [];
+  recipients: RecipientType[] | [];
+};
+
+const Statistics: React.FC<IStatistics> = ({ givers, recipients }) => {
+  const totalGivers: number = givers.length;
+  const totalRecipints: number = recipients.length;
+  let suggestedMatch: number = 0;
+  let confirmedMatch: number = 0;
+  givers.map((giver) => {
+    if (giver.isSuggestedMatch) {
+      if (giver.hasConfirmedMatch) {
+        confirmedMatch++;
+      } else {
+        suggestedMatch++;
+      }
+    }
+  });
```

Comment:
This should have been done with a useEffect hook, with giver as second parameter. Now the givers list is iterated upon every render. The suggestedMatch and confirmedMatch should also be put into a state.
For this to work, the states must be set after the loop has finished. The javascript filter method could probably be used here?